roles/sap_hana_preconfigure/tasks/RedHat/generic/assert-tsx.yml

```diff
@@ -5,15 +5,17 @@
   when:
     - ansible_architecture == 'x86_64'
     - ansible_distribution == 'RedHat'
-    - ansible_distribution_major_version == '8'
+    - (ansible_distribution_major_version >= '9') or
+      (ansible_distribution_major_version == '8' and
+       __sap_hana_preconfigure_fact_ansible_distribution_minor_version | int >= 3)
     - __sap_hana_preconfigure_fact_ansible_distribution_minor_version | int >= 3
```

Comment:
This conflicts with the new line 8?

Reply:
Yes, thanks for catching this! Will fix it soon.